sdk/js/src/svm.ts

```diff
@@ -435,6 +439,10 @@ export async function constructSwapBid(
   const { userToken, searcherToken, user, tokenInitializationConfigs } =
     extractSwapInfo(swapOpportunity);
 
+  if (swapOpportunity.memo) {
```

Comment:
maybe we want to make this backward compatible for limo/submit bid as well?

Reply:
do you mean adding this to limit orders? let's decide if we need to prioritize that.